runtime/themes/base16_default_dark.toml

```diff
@@ -33,11 +33,12 @@
 "namespace" = "base0E"
 "ui.help" = { fg = "base06", bg = "base01" }
 
-"info" = "base03"
-"hint" = "base03"
-"debug" = "base03"
-"diagnostic" = "base03"
-"error" = "base08"
+"info" = { fg = "base03", bg = "base01" }
+"hint" = { fg = "base03", bg = "base01" }
+"debug" = { fg = "base03", bg = "base01" }
```

Comment:
Since all these info/debug/warning/error colors are used in places besides the gutter, I don't recommend setting `bg`. Instead a new UI scope should be added `ui.gutter` that will be used for all the gutters including line numbers. You can add the styling here and I'll patch it in later

Reply:
Okay changed to:
```toml
"ui.gutter" = { bg = "black" }
"info" = "gray"
"hint" = "gray"
"debug" = "gray"
"warning" = "yellow"
"error" = "red"
```
Is this correct?

Reply:
Let's make `info` light blue.

Reply:
Changed `info`:
- `base16_default_dark.toml`: `base0D`
- `base16_default_light.toml`: `base0D`
- `base16_terminal.toml`: `light-blue`
- `base16_theme.toml`: `blue` (instead of `light-blue` because for compatibility for all terminal, the default theme should not contain light variant color)